wgpu-types/Cargo.toml

```diff
@@ -48,6 +48,20 @@ trace = ["std"]
 # Enable web-specific dependencies for wasm.
 web = ["dep:js-sys", "dep:web-sys"]
 
+# Enables the `parking_lot` set of locking primitives.
+# This is the recommended implementation and will be used in preference to
+# any other implementation.
+# Will fallback to a `RefCell` based implementation which is `!Sync` when no
+# alternative feature is enabled.
+parking_lot = ["dep:parking_lot"]
```

Comment:
(as per comment on https://github.com/gfx-rs/wgpu/pull/7746#issuecomment-3003677289 I'd like `std` to directly imply `parking_lot` if possible with no way of having `std` disabled and `parking_lot` still enabled. But let's talk it out on the on the other PR)